mdviewer.yml

```diff
@@ -0,0 +1,16 @@
+
+---
+services:
+  ocis:
+    depends_on:
+      md-viewer:
+        condition: service_completed_successfully
+
+  md-viewer:
```

Comment:
```suggestion
  mdpresentation-viewer:
```

Reply:
or `md-presentation-viewer` to keep the same name as you use on docker hub?

Reply:
whatever you decide, apply it also to the .yml file name

Reply:
we should use `mdpresentation-viewer` for docker image as well to align with the release artifacts and the name in package.json